tests/Util/fixtures/YamlConvert/container_built_differently/comparaison_on_container_built_failed

```diff
@@ -0,0 +1,10 @@
+\*****************************************\
+
+Symfony\Bundle\MakerBundle\Tests\Util\fixtures\YamlConvert\FakeClass\SimpleFakeClassServiceThree:
+    arguments:
+      index_0: '@doctrine.custom_entity_manager'
+
+
+$services->set(SimpleFakeClassServiceThree::class)
+    ->arg(0, ref('doctrine.custom_entity_manager'))
+;
```

Comment:
Should this be deleted?

Reply:
Yes, in a old conversation you told me that the users mustn't using this index_0 case now. If they use, they'll see an exception.